snow/validators/set.go

```diff
@@ -318,20 +318,19 @@ func (s *vdrSet) len() int {
 	return len(s.vdrSlice)
 }
 
-func (s *vdrSet) List() []*Validator {
+func (s *vdrSet) Map() map[ids.NodeID]*GetValidatorOutput {
```

Comment:
```suggestion
// Returns the map with the node ID as a key, and "GetValidatorOutput" as a value.
func (s *vdrSet) Map() map[ids.NodeID]*GetValidatorOutput {
```

?

Reply:
We typically put the comments on the interface rather than the unexported type. This is because when someone is using `Map` editors don't actually know the concrete type - so they show the interface definition.